app/controllers/bulkrax/entries_controller.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require_dependency "bulkrax/application_controller"
+require_dependency "oai"
+require 'fileutils'
+
+module Bulkrax
+  class EntriesController < ApplicationController
```

Comment:
Is line 3 necessary if you're  inheriting from ApplicationController on line 8?

Reply:
@geekscruff Is line 3 necessary?

Reply:
Probably not, I'll have a test.